ethportal-peertest/src/scenarios/gossip.rs

```diff
@@ -299,7 +299,7 @@ pub async fn test_gossip_dropped_with_find_content(peertest: &Peertest, target:
     // send find_content request from fresh target to target
```

Comment:
nit: Unrelated to this pr, but this comment is wrong. Or at least, it looks like it was correct, but than line 301 was  commented out and line 302 is added. Would be nice to fix it if you know what is happening here.

Reply:
I removed it, that seems like an artifact left when writing the test

Reply:
The comment should still be updated to